check.py

```diff
@@ -1,9 +1,16 @@
+from __future__ import annotations
```

Comment:
Don't you want to just update your virtual environment to Python 3.9.x?

Reply:
We claim to support Python 3.7 or newer. You can't have annotations that aren't supported in Python 3.7 unless you postpone their evaluation, which is exactly what this line does.

Reply:
@dakq I guess we can ditch support of Python 3.7 and just pin Python 3.9.x in the project, because it is not too old, not too bleeding edge, just right in the middle. What do you think?

Reply:
Not up to me to decide. I would ask Anna

Reply:
@dakq she agreed to use Python 3.9.x. Could you please make corresponding changes to the PR?